zap-desktop-getting-started.md

```diff
@@ -2,71 +2,72 @@
 
 <p align='center'>
-  <img src='https://i.imgur.com/M3O4jKV.png' alt='screenshot' />
+  <img src='https://imgur.com/eo783Fu.png' alt='screenshot' />
 </p>
 
 ## Step 3: Verify your seed
-Next, Zap will ask you to verify your seed by prompting you to enter three randomly selected words from your seed. Once entered correctly you'll see the inputs turn green and you can proceed. If you've entered the words incorrectly the Zap UI will turn the inputs red. 
+Next, Zap will ask you to verify your seed by prompting you to enter three randomly selected words from your seed. Once entered correctly you’ll see the inputs turn green and you can proceed. If you’ve entered the words incorrectly the Zap UI will turn the inputs red.
 
 <p align='center'>
-  <img src='https://i.imgur.com/5EJu8pH.png' alt='screenshot' />
+  <img src='https://imgur.com/1Q3R0Jy.png' alt='screenshot' />
 </p>
 
 ## Step 4: Wallet password
 Now you will set your wallet password. This password will be used to encrypt your wallet data. The password must be at least eight characters in length. You'll use this password to unlock your wallet on start up. If you lose this password the only way to gain access to your wallet is by way of recovery via your 24 word seed. Please back up this password and keep it safe!
 
 <p align='center'>
-  <img src='https://i.imgur.com/UC1nLrR.png' alt='screenshot' />
+  <img src='https://imgur.com/kMtVYmq.png' alt='screenshot' />
 </p>
 
 ## Step 5: Wallet name
-Now you can create your wallet name. This is new to Zap. Your wallet's name will be used as a "handle" or "username" within your Zap Launchpad to help keep your wallet identifiable and organized now that Zap allows multiple wallets to be created.
+Now you can create your wallet name. Your wallet’s name will be used as a “handle” or “username” within your Zap Launchpad to help keep your wallet identifiable and organized now that Zap allows multiple wallets to be created. The wallet name is not broadcast to the network and is for internal use only.
 
 <p align='center'>
-  <img src='https://i.imgur.com/nnJ1T82.png' alt='screenshot' />
+  <img src='https://imgur.com/MYnpuwl.png' alt='screenshot' />
 </p>
 
-## Step 6: Network
-Now, for the first time, you can select which network you would like to run on. If you select mainnet you will see a yellow warning:
-
-```
-WARNING: You may lose your funds. Mainnet support is experimental and has not been thoroughly tested. Use Testnet to experiment without risk.
-```
-
-Creating mainnet wallets inside of Zap is very risky and intended for developers and advanced users to help test. You may lose funds on mainnet with neutrino, and we encourage those who would like to experiment without risk to use testnet.
-
+## Step 6: Channel Backups
```

Comment:
nit: I think we need to explain to users that their channel backups are encrypted. it looks a little creepy that we have Google and Dropbox options and don't explain that all channel backup data is encrypted and is useless standalone